apps/docs/content/components/Card/react.mdx

```diff
@@ -83,7 +83,7 @@ You can add an `icon` to enhance the visual context. We recommend using an [Octi
 
 ```jsx live
 <Card href="https://github.com">
-  <Card.Icon icon={<CopilotIcon />} color="purple" hasBackground />
+  <Card.Icon icon={CopilotIcon} color="purple" hasBackground />
```

Comment:
Does Icon still support passing the instantiated component, or only reference? I.e. if user wanted to customize values here, can they still do it?

Reply:
No, that was removed by design after speaking with @danielguillan to avoid situations like this:

```tsx
<Card.Icon icon={<CopilotIcon size={20} />} size={44} />
```

Since `Card.Icon` forwards all of its props to the provided icon, you could easily end up in these awkward situations where two props clash. 

Do you think this should be called out as a breaking change in the changeset?

Reply:
Thanks for this example. Makes sense 👍 

This is a breaking change if we don't allow instantiated JSX anymore, yeah. Unless there's a really important reason for it, we should continue to support both ways, as we have _many_ instances in dotcom and elsewhere that use that approach and we can more easily work around it on our side than the users. In many of those instances, the user doesn't apply size too, so this is less of a risk IMO.

E.g.

<img width="557" alt="Screenshot 2024-10-31 at 10 44 51" src="https://github.com/user-attachments/assets/39bebcb6-1dd8-4197-ad74-cf6612f800f1">

Suggestion:

Update our docs and examples to show the reference approach (happy path), and ensure that instantiated JSX can continue to be passed for continuity. How does that sound?

Reply:
Update for posterity.. I've pushed the changes ☝️ in [this commit](https://github.com/primer/brand/pull/798/commits/6acc73575c059c8da2e8f2b55a6027d6e5c50092)

- passing JSX is now working
- removed breaking change notice
- added a new story to track future regressions